content/docs/command-reference/version.md

```diff
@@ -44,6 +44,8 @@ The detail of DVC version depends upon the way of installing DVC.
   [in our core repo](https://github.com/iterative/dvc/blob/master/dvc/version.py).
   For example `0.40.2`.
 
+  > 💡 To only get this number, use `dvc -V`.
```

Comment:
No need to make it a note?

Reply:
I think so, because it's not about `dvc version`.

> Still it feels it belongs here (as a note) more than in the cmd ref index (where `dvc -C` will be, for example).